.cascade-code/Chapman.edu/_components/left_nav/left_nav_children.vtl

```diff
@@ -0,0 +1,137 @@
+## Formerly LeftNavNoSiblings. Used for index pages.
+## New left nav format that works without a block
+
+#set ( $pagePath = $currentPage.path )
+#set ( $parentDir = $currentPage.parentFolder )
+
+## Builds the left navigation
+#macro ( buildLeftNav )
+    #buildLeftNavTitle
```

Comment:
I guess the parentheses aren't needed but I'd prefer macros were called with them. It's a little visual signal that improves readability:

    #buildLeftNavTitle()